src/cudd/adapter.h

```diff
@@ -178,7 +178,7 @@ class cudd_bcdd_adapter : public cudd_adapter
   inline BDD
   apply_imp(const BDD& f, const BDD& g)
   {
-    return f.Ite(g, _mgr.bddOne());
+    return (!f).Or(g);
```

Comment:
Out of curiousity: do you by any chance have any data on this improving performance?

Reply:
No, I did not benchmark it, and sorry, I messed up things in my mind: I was about to write that CUDD does implement the fallback to binary operators, but turns out this is only the case for ZDDs. So there should be no measurable performance difference. Good that you asked, I’ll remove the commit.

Reply:
The only difference I could think of that makes this have an impact is in how the cache is used (since it's now a 3-ary tuple). At the top of my head, I cannot recall the technical details on CUDD's memoization table(s) - so, I cannot say whether there could be some benefit in not polluting the Apply table or some deficit in having larger entries (and hence more cache misses) with a 3-ary entry.